plotly/tools.py

```diff
@@ -15,6 +15,8 @@
 import six
 import requests
 
+import numpy as np
```

Comment:
we can't make `numpy` a dependency. check out how we do "protected" imports of `pandas` to get around this